### PR TITLE
Add a section about linking process automation using rnpm

### PR DESCRIPTION
**Pull request by Kureev**
According to this conversation: https://twitter.com/martinkonicek/status/676698315451408384

### CODE REVIEW

File: docs/LinkingLibraries.md [hunk 1]
```diff
@@ -24,7 +24,41 @@ error as soon as you try to use the library._
 
 ## Here the few steps to link your libraries that contain native code
 
-### Step 1
+### Automatic linking
+
+There is a way to link your native dependencies automatically by using [rnpm](http://github.com/rnpm/rnpm):
```
Comment (skevy) [inline docs/LinkingLibraries.md]: Reword: `"[rnpm](http://github.com/rnpm/rnpm)" is a community project that allows linking of native dependencies automatically:"

**Comment by skevy** [comment]
@Kureev thanks for adding! Will merge after you update.

**Comment by Kureev** [comment]
@skevy done!

**Comment by skevy** [comment]
@facebook-github-bot shipit

**Pull request merged**